wordpress-mobile/WordPress-Android
# Issue/6508 google login callbacks

### Fix
Add Google API calls and callbacks to the email screen as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6508.

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Tap Google login view.
3. Choose Google account.
4. Notice messages in `logcat`.
```
org.wordpress.android.beta D/WordPress-API: Dispatching action: AccountAction-PUSH_SOCIAL_LOGIN
```
```
org.wordpress.android.beta E/WordPress-API: LoginEmailFragment.onSocialChanged: UNKNOWN_USER - Hmm, we can't find a WordPress.com account with this username and password combo. Please double check your information and try again.
```
```
org.wordpress.android.beta E/WordPress-API: LoginEmailFragment.onSocialChanged: USER_EXISTS - This social account is already associated with an existing WordPress.com account.
```

#### Note
In order to hit the `OnAuthenticationChanged` callback , the Google account selected must already be connected to WordPress.  You can connect/disconnect a social account to/from a WordPress account at https://wordpress.com/me/security/social-login/ on the web.  If it is connected, you can set a breakpoint [here](https://github.com/wordpress-mobile/WordPress-Android/pull/6671/files#diff-c0e31e11f2dbf99904de706676fa1602R433).  If it's not connected, you'll receive the `USER_EXISTS ` error message in `OnSocialChanged`.
